.github/pull_request_template.md

```
@@ -1,48 +1,43 @@
 
-List the checks you ran and the exact commands when useful.
+<!-- Approaches tried and abandoned during original development. -->
 
-```bash
-# commands here
-```
+<details>
+<summary>Diff Statistics</summary>
 
-## Impact
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)